js/api_openlayers.js

```diff
@@ -352,6 +352,41 @@ function displayTrack(xml, update) {
   }
 }
 
+function displayAllUsers(xml, update) {
+  altitudes = {};
+  var totalMeters = 0;
+  var totalSeconds = 0;
+  var points = [];
+  var positions = xml.getElementsByTagName('position');
+  var posLen = positions.length;
+  for (var i = 0; i < posLen; i++) {
+    var p = parsePosition(positions[i], i);
+    totalMeters += p.distance;
+    totalSeconds += p.seconds;
+    p['totalMeters'] = totalMeters;
+    p['totalSeconds'] = totalSeconds;
+    // set marker
+    setMarker(p, i, posLen, true);
+    // update polyline
+    var point = ol.proj.fromLonLat([p.longitude, p.latitude]);
+    points.push(point);
+  }
+  var extent = layerTrack.getSource().getExtent();
```

Comment:
Extent calculation probably doesn't work as the function does not update layerTrack.